dask_glm/tests/test_algos_families.py

```diff
@@ -8,9 +8,33 @@
 
 from dask_glm.algorithms import (newton, lbfgs, proximal_grad,
                                  gradient_descent, admm)
-from dask_glm.families import Logistic, Normal, Poisson
+from dask_glm.families import Family
 from dask_glm.regularizers import Regularizer
 from dask_glm.utils import sigmoid, make_y
+from unittest.mock import patch
```

Comment:
@TomAugspurger yea the point was just to make sure that if someone changes this code, the tests will catch it since it would break every subclass would potentially break.